tox.ini

```diff
@@ -2,13 +2,12 @@
 min_version = 4.0
 
 envlist =
-    python3.10-django3.2-wagtail5.2-postgres15
-    python{3.8,3.9,3.10,3.11,3.12}-django{4.2}-wagtail{5.2,6.0}-postgres15
-    python{3.10,3.11,3.12}-django{5.0}-wagtail{5.2,6.0}-postgres15
+    python{3.8,3.9,3.10,3.11,3.12}-django4.2-wagtail{5.2,6.0,6.1,6.2}-postgres15
+    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0,6.1,6.2}-postgres15
```

Comment:
```suggestion
    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{5.2,6.1,6.2}-postgres15
    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.1,6.2}-postgres15
    python{3.12}-django5.1-wagtail{6.2}-postgres15
```

Reply:
Updated in f647ac7

Reply:
I suppose this means Python 3.8 support is dropped, since it eaches EOL on 2024-10-31?

I have therefore also updated the README, CHANGELOG, GitHub Actions and Trove classifiers in 763a50c